encord/client.py

```diff
@@ -511,19 +511,43 @@ def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_model
 
         return self._querier.basic_setter(ProjectCopy, self._config.resource_id, payload=payload)
 
-    def get_label_row(self, uid: str, get_signed_url: bool = True) -> LabelRow:
+    def get_label_row(
+        self,
+        uid: str,
+        get_signed_url: bool = True,
+        *,
+        include_object_feature_hashes: Optional[typing.Set[str]] = None,
+        include_classification_feature_hashes: Optional[typing.Set[str]] = None,
+    ) -> LabelRow:
         """
         This function is documented in :meth:`encord.project.Project.get_label_row`.
         """
-        payload = {"get_signed_url": get_signed_url, "multi_request": False}
+        payload = {
+            "get_signed_url": get_signed_url,
+            "multi_request": False,
+            "include_object_feature_hashes": optional_set_to_list(include_object_feature_hashes),
```

Comment:
What error do we get if we try to pass a set in?

Reply:
Sets cannot be used for JSON objects.

Reply:
So the converstion to JSON when creating the HTTP payload fails.